CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fixed import cycle when importing async helpers ([#311](https://github.com/opensearch-project/opensearch-py/pull/311))
 - Fixed userguide for async client ([#340](https://github.com/opensearch-project/opensearch-py/pull/340))
 - Include parsed error info in TransportError in async connections (fixes #225) ([#226](https://github.com/opensearch-project/opensearch-py/pull/226)
+- Enhanced existing API generator to use the OpenAPI spec as the input format ([#412](https://github.com/opensearch-project/opensearch-py/pull/412)
```

Comment:
We're replacing an ES spec by an OpenSearch one, so this should probably say "Use OpenSearch OpenAPI spec"